### DISCUSSION TITLE
how to /replacenear all blocks on y=100?

### DISCUSSION THREAD

**Discussion by maksim10411072** [IntellectualSites/FastAsyncWorldEdit#2036] [Q&A] [answered]
i need to replace all blocks on y=100. How to do it?

**Comment by PierreSchwang** [comment, marked as answer]
Use a global mask using an expression mask (`//gmask =y==100`).

Further infos: 
- https://worldedit.enginehub.org/en/latest/usage/general/masks/#expression-mask
- https://worldedit.enginehub.org/en/latest/usage/other/expressions/
- https://worldedit.enginehub.org/en/latest/commands/
